v4/source/introduction.yaml

```diff
@@ -214,23 +214,205 @@ tags:
 
       The list of Mattermost WebSocket events are:
       - added_to_team
+        ```
+        {
+          event: 'added_to_team',
+          data: {
+            team_id: '<team_id>',
+            user_id: '<user_id>'
+          },
+          broadcast: {
+            omit_users: null,
+            user_id: '<user_id>',
+            channel_id: '<channel_id>',
+            team_id: '<team_id>'
+          },
+          seq: '<seq_id>'
+        }
+        ```
       - authentication_challenge
-      - channel_converted
       - channel_created
+        ```
+        {
+          'event': 'channel_created',
+          'seq'  : '<seq_id>',
```

Comment:
Can we make sure the ordering for `seq` is the same for all events for consistency's sake? Here it appears after `event` but in all the others it appears after `broadcast`